src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +20,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(s -> s.split(","))
+                .flatMap(Arrays::stream)
+                .map(Integer::parseInt)
```

Comment:
```suggestion
                .mapToInt(Integer::parseInt)
```
if you use IntStream, you don't need Comparator in .min()